ci2/ci/ci.py

```diff
@@ -75,19 +75,33 @@
     config = {}
     config['number'] = pr.number
     if pr.batch:
-        status = await pr.batch.status()
-        for j in status['jobs']:
-            if 'duration' in j and j['duration'] is not None:
-                j['duration'] = humanize.naturaldelta(datetime.timedelta(seconds=j['duration']))
-            attrs = j['attributes']
-            if 'link' in attrs:
-                attrs['link'] = attrs['link'].split(',')
-        config['batch'] = status
+        config['batch'] = await prepare_batch_status(pr.batch)
         config['artifacts'] = f'{BUCKET}/build/{pr.batch.attributes["token"]}'
+    if pr.most_recent_build is not None:
+        config['most_recent_build'] = {
+            'source_sha': pr.most_recent_build.source_sha,
+            'target_sha': pr.most_recent_build.target_sha,
+            'build_state': pr.most_recent_build.build_state,
+            'batch': await prepare_batch_status(pr.most_recent_build.batch),
+            'artifacts': f'{BUCKET}/build/{pr.most_recent_build.batch.attributes["token"]}'
+        }
+    else:
+        config['most_recent_build'] = None
 
     return config
 
 
+async def prepare_batch_status(batch):
```

Comment:
Can we get a batter name for this?  Jinja2 calls the render parameters "context", maybe `batch_status_template_context` or something?